specification.md

```diff
@@ -166,7 +162,7 @@ using either a remote or local job management library; application jobs
 are then submitted to the pilot system, which sends them directly to the
 existing pilot job instances for execution, bypassing queuing
 systems/LRMs. The requirements for the APIs used to submit the pilot jobs
-as well as those used to run the application and remote job management
+as well as those used to run the application l and remote job management
```

Comment:
l?

Reply:
Well, the revert also covers some small bug fixes :-/

Reply:
Hah, OK. I think we can safely keep this fix though.

Reply:
ok, done.